tests/Scripting/TasksDate.test.ts

```diff
@@ -105,3 +105,54 @@ describe('TasksDate', () => {
         expect(new TasksDate(moment('2023-02-31')).fromNow.groupText).toEqual('%%320230611%% Invalid date');
     });
 });
+
+describe('TasksDate - postpone', () => {
+    beforeEach(() => {
+        jest.useFakeTimers();
+        jest.setSystemTime(new Date('2023-11-28'));
+    });
+
+    afterEach(() => {
+        jest.useRealTimers();
+    });
+
+    it('should postpone an older date (before yesterday) to today', () => {
+        const tasksDate = new TasksDate(moment('2023-11-20'));
+
+        const postponedDate = new TasksDate(tasksDate.postpone());
+
+        expect(postponedDate.formatAsDate()).toEqual('2023-11-28');
+    });
+
+    it('should postpone yesterday date to today', () => {
+        const tasksDate = new TasksDate(moment('2023-11-27'));
+
+        const postponedDate = new TasksDate(tasksDate.postpone());
+
+        expect(postponedDate.formatAsDate()).toEqual('2023-11-28');
+    });
+
+    it('should postpone today date to tomorrow', () => {
+        const tasksDate = new TasksDate(moment('2023-11-28'));
+
+        const postponedDate = new TasksDate(tasksDate.postpone());
+
+        expect(postponedDate.formatAsDate()).toEqual('2023-11-29');
+    });
+
+    it('should postpone tomorrow date to after tomorrow', () => {
+        const tasksDate = new TasksDate(moment('2023-11-29'));
+
+        const postponedDate = new TasksDate(tasksDate.postpone());
+
+        expect(postponedDate.formatAsDate()).toEqual('2023-11-30');
+    });
+
+    it('should postpone a future date (after tomorrow)', () => {
+        const tasksDate = new TasksDate(moment('2024-03-20'));
+
+        const postponedDate = new TasksDate(tasksDate.postpone());
+
+        expect(postponedDate.formatAsDate()).toEqual('2024-03-21');
+    });
```

Comment:
Nice choice of tests....

The blank lines moving the date values apart makes it harder to skim-read, but if the code is retained, the solution to that is to quickly extract a helper function to remove the repetition of code.

Reply:
I'm happy to merge this as-is, to keep this PR localised...

Reply:
> Do you think it is worth an effort (I can do it), or we shall move the logic and the tests to Postponer.ts later and this will be removed anyways?

In the interests of both my time and yours, please can we discuss this in a future pairing session before deciding and changing the code....